plugins/catalog-react/src/hooks/index.ts

```diff
@@ -16,4 +16,4 @@
 export { EntityContext, useEntity, useEntityFromUrl } from './useEntity';
 export { useEntityCompoundName } from './useEntityCompoundName';
 export { useRelatedEntities } from './useRelatedEntities';
-export { useStarredEntities } from './useStarredEntities';
+export { isStarredEntity, useStarredEntities } from './useStarredEntities';
```

Comment:
- This was added so I could use it as a pure function within the reducer filtering logic which is further downstream then the top level `useEntityListState` hook that wraps all of that. It might be possible to treat this as a side effect to the action though. I've avoided needing something like thunks here so far but I think it will be necessary to have the filter actions trigger side effects (especially when it comes to severFilters). This will yet require introducing a new state pattern. Any preferences to the style of reducer side effects?